src/query.c

```diff
@@ -260,7 +265,7 @@ QueryNode *NewFuzzyNode_WithParams(QueryParseCtx *q, QueryToken *qt, int maxDist
   QueryNode *ret = NewQueryNode(QN_FUZZY);
   q->numTokens++;
 
-  if (qt->type == QT_TERM) {
+  if (qt->type == QT_TERM || qt->type == QT_NUMERIC || qt->type == QT_SIZE) {
```

Comment:
In another PR (on master) we can use  `rm_strndup` for `QT_NUMERIC` and `QT_SIZE`